<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>604</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>604</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>368</width>
      <height>109</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="text_1">
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(255, 255, 255);
font: 20pt &quot;Segoe Script&quot;;</string>
       </property>
       <property name="text">
        <string>Приветствую тебя!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="text_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 18pt &quot;Segoe Script&quot;;
</string>
       </property>
       <property name="text">
        <string>    Назови свое имя</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="L_name">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>231</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="B_ok">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>360</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>ок</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
